src/stress/shaders/entry_points.spec.ts

```diff
@@ -0,0 +1,12 @@
+export const description = `
+Stress tests covering behavior around shader entry points.
+`;
+
+import { makeTestGroup } from '../../common/framework/test_group.js';
+import { GPUTest } from '../../webgpu/gpu_test.js';
+
+export const g = makeTestGroup(GPUTest);
+
+g.test('many_entry_points')
+  .desc(`Tests compilation and usage of shaders with a huge number of entry points.`)
```

Comment:
There could end up being a normative limit to the number of entry points, in which case this would become a validation test instead. Let's leave a note (`TODO:`) so whoever looks at this knows to check.
cc @alan-baker

Reply:
Done